projects/i18n/package.json

```diff
@@ -1,6 +1,14 @@
 {
   "name": "@skyux/i18n",
   "version": "5.0.0-beta.2",
+  "schematics": "./schematics/collection.json",
+  "scripts": {
+    "test": "../../node_modules/.bin/nyc npm run test:unit",
+    "test:unit": "../../node_modules/.bin/ts-node --project tsconfig.schematics.json ../../node_modules/jasmine/bin/jasmine.js --config=jasmine.json"
+  },
```

Comment:
Just for my own education... The Angular docs suggest adding several scripts here to make sure your schematics source files get compiled into the library bundle. Is that what `scripts/build-schematics.js` is doing? Why go that route instead?

Reply:
The scripts that Angular provides only work on Windows; I also wanted to be able to run the schematics tests in our GitHub Actions workflow, so I created separate scripts for that end.